data/moves.js

```diff
@@ -17190,10 +17181,7 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, mystery: 1},
 		onTryHit(target) {
```

Comment:
`onTryImmunity`?

I think you missed Trick too.

Reply:
Whoops, I had it in my testing tree, just forgot to transfer it to my clean pull. Sorry about that. But hey, that reminded me to add Endeavo(u)r, which is also supposed to check the target's HP before accuracy.